pages/consoles.html

```diff
@@ -0,0 +1,420 @@
+		font-weight: 500;
+		transition: background-color 0.2s ease;
+		width: 100%;
+		text-align: center;
+		margin-top: 16px;
+	}
+
+	.vendor-button:hover {
+		background-color: color-mix(in srgb, var(--primary-color) 90%, white 3%);
+		color: white;
+		text-decoration: none;
+	}
+
+	.cards-container {
+		display: grid;
+		justify-content: center;
+		column-gap: 24px;
+		row-gap: 24px;
+		grid-template-columns: 1fr 1fr;
+		margin-top: 24px;
+	}
+	@media (max-width: 800px) {
+		.cards-container {
+			grid-template-columns: 1fr;
+		}
+	}
+	.add-your-company {
+		margin-top: 24px;
+		margin-bottom: 42px;
+	}
+	.add-your-company .card-content {
+		padding: 20px;
+		text-align: left;
+	}
+
+</style>
+
+<div class="consoles-hero">
+	<h2>Console support in Godot</h2>
+</div>
+
+<div class="container">
+	<div class="consoles-wrapper">
+		<h2>Godot and Console Porting</h2>
+		<div class="cards-container">
+			<div class="content-card">
+				<h4>Closed ecosystems</h4>
+				<p>
+					Console platforms are like closed ecosystems. Working with them
+					requires special permissions, SDKs, and developer hardware. These
+					platforms are governed by strict NDAs and legal requirements.
+				</p>
+				<p>
+					While Godot cannot directly provide official support due to its open
+					source license, developers approved by console vendors can still build
+					and ship console games using Godot with the help of certified
+					third-party providers.
+				</p>
+			</div>
+			<div class="content-card">
+				<h4>An open source system</h4>
+				<p>
+					Godot is fully open source and licensed under MIT. That means no NDAs,
+					no closed tools, and no legal liability accepted. Console development,
+					however, requires exactly the opposite: legal agreements, restricted
+					access, and liability coverage.
+				</p>
+				<p>
+					Because of this, the Godot Foundation has decided not to develop or
+					maintain official console ports, doing so would compromise its core
+					values of transparency and freedom.
+				</p>
+			</div>
+		</div>
+
+		<div class="cards-container">
+			<div class="content-card">
+				<h4>Console porting in Godot</h4>
+				<p>
+					You can release a Godot game on consoles. You'll need to be approved
+					by the platform holder and either create the export tools yourself or
+					use middleware from a certified third party.
+				</p>
+				<p>
+					The Godot engine itself can run on consoles, but export templates for
+					those platforms must be built separately. These templates are
+					typically created using official SDKs, and thus can only be
+					distributed privately to developers approved by the console makers.
+					Several third-party companies offer this middleware under the
+					necessary legal constraints.
+				</p>
+				<p>
+					For more information, check the following link:
+					<a
+						href="https://godotengine.org/article/about-official-console-ports/"
+					>
+						official console ports</a
+					>.
```

Comment:
```suggestion
					For more information, check the following link:
					<a
						href="https://godotengine.org/article/about-official-console-ports/"
					>official console ports</a>.
```
Just to make sure that no stray spaces get underlined.